content/blog/2021-08-24-transfer-learning-experiments.md

```diff
@@ -14,6 +14,7 @@ tags:
   - Experiments
   - Reproducibility
   - DVC
+  - Pre-trained Models
```

Comment:
why changing this here?

Reply:
I think I was just updating the tag to match the topic better. I can remove it if you think it's unnecessary.